posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -340,15 +397,17 @@ def to_path_bounce_query(self) -> ast.SelectQuery:
 LEFT JOIN (
     SELECT
         breakdown_value,
-        avg(is_bounce) AS bounce_rate
+        coalesce(avgIf(is_bounce, current_period_segment), 0) AS bounce_rate,
```

Comment:
Do we need the coalesce to zero? IMO returning NULL is more accurate

Reply:
I've changed this back to returning `NULL`